packages/loot-core/src/client/shared-listeners.ts

```diff
@@ -3,9 +3,22 @@ import { t } from 'i18next';
 
 import { listen, send } from '../platform/client/fetch';
 
+import {
+  addNotification,
+  closeAndDownloadBudget,
+  getAccounts,
+  getCategories,
+  getPayees,
+  loadPrefs,
+  pushModal,
+  resetSync,
+  sync,
+  uploadBudget,
+} from './actions';
 import type { Notification } from './state-types/notifications';
+import { type AppStore } from './store';
 
-export function listenForSyncEvent(actions, store) {
```

Comment:
Remove actions from `useActions` here. Instead, we manually dispatch the actions through the store's dispatch property.